### ISSUE TITLE
Improving Admin.py of every App

### ISSUE THREAD

**Issue by shriyanka** [ncs-jss/HTTP_200#52]
As discussed with @DESHRAJ  we will be creating a smooth admin interface for all apps so that system administrators do not find it difficult to add new entry in the database.
This is as per current discussions we can change it or see what is required for version v.2 onwards.

Currently assigning it to my self. If any one else is interested to work on it feel free to leave a comment.

**Comment by shriyanka** [comment]
TODO : Create custom templates by overriding Admin templates

**Comment by Shivam1904** [comment]
@DESHRAJ  @shriyanka - What does "smooth interface" mean ?
Do we have to change the front-end of the panel or any functionality ?

**Comment by DESHRAJ** [comment]
@shriyanka django 1.9 has quite decent interface for django admin. So, I think porting to django 1.9 can solve the issue.

**Comment by shriyanka** [comment]
@deshraj i haven't seen the interface yet but no prob I will tweak with it. 
however i think you have downgraded the app version to 1.8 so whats up with it??

**Comment by DESHRAJ** [comment]
@shriyanka Yes I have downgraded since there were conflicts with the feeds app that  needs to be fixed and I was working on building the features. So, I thought to do it later.
